geonode/layers/models.py

```diff
@@ -421,6 +421,18 @@ class Attribute(models.Model):
         default=True)
     display_order = models.IntegerField(_('display order'), help_text=_(
         'specifies the order in which attribute should be displayed in identify results'), default=1)
+    
+    searchable = models.BooleanField(
```

Comment:
this stuff must be outside of GeoNode. Let's create a WMLayer model in wm_exra with a OneToOneField field to Layer

Reply:
In my case, this field is necessary to work dataverse migrations because it's referred in a couples of columns.